freshmaker/handlers/internal/update_db_on_odcs_compose_fail.py

```diff
@@ -50,6 +50,9 @@ def handle(self, event):
             Compose.odcs_compose_id == event.compose["id"],
             ArtifactBuildCompose.compose_id == Compose.id)
 
+        if builds_with_compose:
+            self.log_error("ODCS compose %s failed", event.compose["id"])
+
```

Comment:
And here it makes sense to log error only when there are builds waiting for this compose.